app/views/admin/_msa_component.html.erb

```diff
@@ -0,0 +1,44 @@
+<%= link_to "Create new MSA component", new_msa_component_path, class: "govuk-button"  %>
+
+<% components.reverse.each do |component| %>
+  <table class="govuk-table">
+    <caption class="govuk-table__caption" id="MsaComponent/<%= component.id %>">MSA component ID: <%= component.id %></caption>
+    <tbody class="govuk-table__body">
+      <tr class="govuk-table__row">
+        <th scope="row" class="govuk-table__header">Name</th>
+        <td class="govuk-table__cell"><%= component.name %></td>
+      </tr>
+      <tr class="govuk-table__row">
+        <th scope="row" class="govuk-table__header">Entity ID</th>
+        <td class="govuk-table__cell"><%= component.entity_id %></td>
+      </tr>
+      <tr class="govuk-table__row">
+        <th scope="row" class="govuk-table__header">Team ID</th>
+        <td class="govuk-table__cell"><%= link_to(component.team_id, "/admin#teams/#{component.team_id}") if component.team_id %></td>
+      </tr>
+      <tr class="govuk-table__row">
+        <th scope="row" class="govuk-table__header">Environment</th>
+        <td class="govuk-table__cell"><%= component.environment %></td>
```

Comment:
`titleize`?

Reply:
I prefer not to, as a dev/admin I'd like to see the raw values in case there's a (casing) bug and `titleize` would hide it.